recipe/meta.yaml

```
@@ -26,7 +24,7 @@ requirements:
   run:
     - python
     - ipykernel >=5.1.2
-    - ipywidgets >=7.0.0
+    - ipywidgets >=7,<8
     - jinja2 >=2.10.1
     - traitlets >=4.3.2
     - {{ pin_compatible('numpy') }}
```

```suggestion
    - numpy >=1.16.4
```
https://inspector.pypi.io/project/pydeck/0.8.0/packages/a7/86/f7478f621e998b36afa4e668088c407b04182e1080d609f6b8d0b1396401/pydeck-0.8.0.tar.gz/pydeck-0.8.0/setup.py#line.218